ca/config.go

```diff
@@ -124,10 +124,15 @@ func SigningPolicy(certExpiry time.Duration) *cfconfig.Signing {
 		certExpiry = DefaultNodeCertExpiration
 	}
 
+	notBefore := time.Now().Round(time.Minute).Add(CertDefaultBackdate).UTC()
+	notAfter := time.Now().Round(time.Minute).Add(certExpiry).UTC()
```

Comment:
Probably a good idea to only call `time.Now()` once to ensure consistency between these timestamps, in case something crazy happens to the clock between the two calls.

It will probably never issue in practice, but I'd rather have one less corner case to think about.